src/Bundle/Doctrine/ORM/DataSource.php

```diff
@@ -65,7 +83,7 @@ public function getExpressionBuilder(): ExpressionBuilderInterface
     public function getData(Parameters $parameters)
     {
         // Use output walkers option in DoctrineORMAdapter should be false as it affects performance greatly. (see #3775)
```

Comment:
I would move this comment to construct docblock & probably we should write a short doc about it